### PR TITLE
Allow field addition for specified dates in bigquery_etl_query

### PR DESCRIPTION
**Pull request by relud**
blocks https://github.com/mozilla/bigquery-etl/pull/1170

### CODE REVIEW

**Comment by scholtzan** [comment]
It's already set for some queries: https://github.com/mozilla/bigquery-etl/blob/f267e38f14769d78ffc04e483904da378b483a1c/sql/firefox_accounts_derived/fxa_auth_bounce_events_v1/metadata.yaml#L11

**Comment by jklukas** [comment]
> It's already set for some queries via arguments

And the fxa_events tables are a bit of a special case where we're doing a `SELECT *` from a table in another project that we have no control over, so we always want to be able to automatically pull in new fields.

I can see the extra safety of specifying certain dates for allowing field addition, but I do worry it will be a bit of a pain when making a PR. If review takes a few days, it will be necessary to update the configured dates before merging. It's definitely a better situation than requiring manual updates to the schema via the BQ console, which is error prone.

File: dags/utils/gcp.py [hunk 1]
```diff
@@ -500,6 +503,10 @@ def bigquery_etl_query(
         + ["--dataset_id=" + dataset_id]
         + (["--project_id=" + project_id] if project_id else [])
         + ["--parameter=" + parameter for parameter in parameters]
+        + [
+            "--schema_update_option={{ 'ALLOW_FIELD_ADDITION' if ds in %r else '' }}"
+            % (allow_field_addition_dates,)
+        ]
```
Comment (jklukas) [inline dags/utils/gcp.py:509]: Can we wrap this in `([ ...] if allow_field_addition_dates else [])`?

Comment (relud) [reply dags/utils/gcp.py:509]: yeah

**Pull request merged**